templates/todo/projects/csharp-cosmos-sql-func/infra/resources.bicep

```diff
@@ -0,0 +1,307 @@
+param location string
```

Comment:
You'll want to move bicep files to the new .repo folder.  SEe the nodejs-mongo project for example.  You'll also need to update the repo.yaml file.  Use that project as a base...

Reply:
Resolved in latest commit.